lib/libc/gen/getpagesize.c

```diff
@@ -1,4 +1,4 @@
-#include <sys/vm.h>
+#include <machine/vm_param.h>
```

Comment:
`machine/vm_param.h` is already included in `sys/vm.h`.

Reply:
I remember I had some problems with this header, but I restored the old version